playbook/roles/nginx_installation/tasks/nginx_ingressController.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Installation of nginx Ingress Controller
+  ansible.builtin.command: kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.1/deploy/static/provider/baremetal/deploy.yaml
+
+- name: Configure the nginx Ingress Controller as the default one
+  ansible.builtin.command: kubectl annotate ingressclass nginx ingressclass.kubernetes.io/is-default-class=true
+
+- name: Pause for installation of Ingress Controller
```

Comment:
Why do we have to wait for 60 seconds?

Reply:
For our old configuration was needed, now it is not and we forgot to change it, we will do that.